src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
+  --color: #00acdc;
+
+  font-family: Roboto, serif;
```

Comment:
Consider using a more specific font stack for better fallback options. For example, 'Roboto, Arial, sans-serif' would provide a more consistent appearance across different browsers and systems.